### PR TITLE
Feature disable buttons by employee type

### PR DESCRIPTION
**Pull request by jacobevans1**
- Adds functionality to disable buttons based on the currently logged in user's employee type.
- Adds functionality to display the currently logged in user's information.
- Fixes several ui and code bugs.

### CODE REVIEW

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/AdminPage.java [hunk 1]
```diff
@@ -56,10 +56,15 @@ public class AdminPage {
 	private Button removeButton;
     
     @FXML
-	private Text employeeFullNameLabel;
-
+	private Text fullnameLabel;
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/AdminPage.java:59]: Fix Camel Casing with to be consistent with the other fields

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java [hunk 2]
```diff
@@ -149,17 +153,22 @@ public class InventoryPage {
...
-	private Text employeeIdLabel;
-
 	@FXML
 	private Text employeeRoleLabel;
 
 	@FXML
 	private TabPane inventoryTreeView;
+	
+	@FXML
+	private Text fullnameLabel;
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java:163]: Fix Camel Casing with to be consistent with the other fields

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/AdminPage.java [hunk 3]
```diff
@@ -83,6 +88,13 @@ private void setupProductButtons() {
 				.bind(Bindings.isNull(this.usersTableView.getSelectionModel().selectedItemProperty()));
 	}
 	
+	private void setCurrentUserSummary() {
+		LocalEmployeeCredentials user = Main.getLoggedInEmployee();
+		this.fullnameLabel.setText(user.getLastName() + COMMA_SEPERATION + user.getFirstName());
+		this.employeeIdLabel.setText(user.getEmployeeID().toString());
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/AdminPage.java:94]: "getEmployeeID" returns a string, there's no need to call "toString()".

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/HomePage.java [hunk 4]
```diff
@@ -68,6 +82,19 @@ private void bindToViewModel() {
...
+	private void setPermissions() {
+		if (!this.viewModel.isManager()) {
+			this.adminButton.setDisable(true);
+		}
+	}
+	
+	private void setCurrentUserSummary() {
+		LocalEmployeeCredentials user = Main.getLoggedInEmployee();
+		this.fullnameLabel.setText(user.getLastName() + COMMA_SEPERATION + user.getFirstName());
+		this.employeeIdLabel.setText(user.getEmployeeID().toString());
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/HomePage.java:94]: "getEmployeeID" returns a string, there's no need to call "toString()".

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java [hunk 5]
```diff
@@ -169,54 +178,73 @@ public class InventoryPage {
...
 		this.setupComponentButtons();
 		this.setupProductButtons();
+		this.setPermissions();
+		this.setCurrentUserSummary();
+	}
+	
+	private void setCurrentUserSummary() {
+		LocalEmployeeCredentials user = Main.getLoggedInEmployee();
+		this.fullnameLabel.setText(user.getLastName() + COMMA_SEPERATION + user.getFirstName());
+		this.employeeIdLabel.setText(user.getEmployeeID().toString());
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java:196]: "getEmployeeID" returns a string, there's no need to call "toString()".

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/OrderPage.java [hunk 6]
```diff
@@ -103,7 +112,16 @@ void initialize() {
...
 
+		this.setPermissions();
 		this.fulfillButton.disableProperty().bind(shouldDisableFulfillButton);
+		this.setCurrentUserSummary();
+	}
+	
+	private void setCurrentUserSummary() {
+		LocalEmployeeCredentials user = Main.getLoggedInEmployee();
+		this.fullnameLabel.setText(user.getLastName() + COMMA_SEPERATION + user.getFirstName());
+		this.employeeIdLabel.setText(user.getEmployeeID().toString());
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/OrderPage.java:123]: "getEmployeeID" returns a string, there's no need to call "toString()".

**Comment by jacobevans1** [comment]
Changes fixed

**Comment by JasonNunez** [comment]
LGTM!

**Pull request merged**